### PR TITLE
Resolve conflicts migrate repo

### PR DESCRIPTION
**Pull request by SeaBlooms**
tested all methods and pytests, added examples.py, bumped version and updated maintainer details

### CODE REVIEW

File: jupiterone/client.py [hunk 1]
```diff
@@ -105,25 +107,55 @@ def _execute_query(self, query: str, variables: Dict = None) -> Dict:
                     raise JupiterOneApiError(content.get('errors'))
                 return response.json()
 
-        elif response.status_code in [429, 500]:
+        elif response.status_code == 401:
+            raise JupiterOneApiError('JupiterOne API query is unauthorized, check credentials.')
+
+        elif response.status_code in [429, 503]:
```
Comment (VDubber) [inline jupiterone/client.py:113]: should 500 also be included?

File: setup.py [hunk 2]
```diff
@@ -8,13 +8,13 @@
 ]
 
 setup(name='jupiterone',
-      version='0.1.0',
+      version='0.3.0',
```
Comment (VDubber) [inline setup.py:11]: Are there any breaking changes? I feel like there are. Can you confirm?

Comment (SeaBlooms) [reply setup.py]: bumped to 1.0.0

File: jupiterone/client.py (hunk 1 above)
Comment (SeaBlooms) [reply jupiterone/client.py:114]: added catch for 500s

**Review by VDubber — APPROVED** [review]

**Review by VDubber — APPROVED** [review]

**Pull request merged**